### DISCUSSION TITLE
Install an specific Version of Pimcore on my Server

### DISCUSSION THREAD

**Discussion by jvolkmib** [pimcore/pimcore#9469] [Q&A / Support]
Hey Guys, 
i have the following Problem: For an Client-Project i need an Development Environment with Version 6.9.4 from Pimcore. 

When i install Pimcore over the normal Way (Pimcore/skeleton), i get Pimcore X. 

After Search over Google i get the Following Solution: 

Get Pimcore over: 
`1. COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton my-project`

`2. composer req pimcore/pimcore:6.2.3
`
(Solution from Bernhard on: https://talk.pimcore.org/t/installation-of-specific-pimcore-version/2948)

Now i trying to install with: 
./vendor/bin/pimcore-install

I get some Errors (Class not found, App/Config not found). 

Is that the right way ? 

B.R. 
Johannes

**Comment by APochmann** [comment]
Although this question is half a year old and you will have figured out in the meantime here some statements from my side as I ran into the same problem and could also not find a soluion somewhere documented:

The solution in the linked description (from 2019) does not work for downgrades to Pimcore releases before Pimcore X. Starting with Pimcore X there was a switch to Symfony Flex and therefore a change of the folder structure was done. So if you install the pimcore skeleton for Pimcore release >=10 the folder structure does not fit for Pimcore release <10 (e.g. there is no app folder anymore).

You could install the newest skeleton for Pimcore V6.9.6 (which is skeleton release 2.8.22) by running

`composer create-project pimcore/skeleton:2.8.22 my-project`

and then downgrade to required pimcore version, e.g.:

`composer req pimcore/pimcore:6.2.3`
